src/app/js/public/search/SearchResultSort.js

```diff
@@ -1,42 +1,62 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
 
 import { field as fieldProptypes } from '../../propTypes';
 import SortButton from '../../lib/components/SortButton';
 
+const getSortableFields = (fields, fieldNames) => {
```

Comment:
The function is too complex, it can be something like that (it is a draft): 

const sortFields = [fieldNames.title, fieldNames.description, fieldNames.detail1, fieldNames.detail2];

return fields
	.filter(field => sortFields.include(field.name)
	.sort(function (a, b) {
		return sortFields.indexOf(a) - sortFields.indexOf(b)
	});